TestModels/SimpleTypes/SimpleBlob/Makefile

```diff
@@ -17,4 +17,18 @@ SERVICE_DEPS_SimpleBlob :=
 SMITHY_DEPS=dafny-dependencies/Model/traits.smithy
 
 # This project has no dependencies 
-# DEPENDENT-MODELS:= 
+# DEPENDENT-MODELS:=
+
+# Constants for languages that drop extern names (Python, Go)
```

Comment:
Will all of these be required? That would really hurt smithy-dafny UX pretty badly.

Reply:
Yes, all are required.
Chatted offline, I think I alleviated your concerns, but let me know if you still have concerns.